_pages/infrastructure/federalist.md

```diff
@@ -8,7 +8,7 @@ Use Federalist for publishing static sites. See [the Federalist homepage](https:
 ## Cost
 
 * **Internal:** Likely free, but start by checking with #federalist with your use case.
-* **External:** Federalist is currently offered for $24949/year. Check in with #federalist-business about your sales lead!
+* **External:** Federalist is currently offered for $24,949/year. Check in with #federalist-business about your sales lead!
```

Comment:
Maybe link to the Federalist pricing page so you don't have to keep this number updated?

Reply:
Done!